api/src/main.ts

```diff
@@ -13,13 +13,12 @@ import { User } from "users/model/users.entity";
     const logger = new Logger("Main");
     dotenvExpand.expand(dotenv.config({ path: ".env" }));
 
-    const app = await NestFactory.create(AppModule);
+    const app = await NestFactory.create(AppModule, { cors: true });
```

Comment:
?????????

Reply:
This is for enable CORS
https://github.com/expressjs/cors#configuration-options

Reply:
Yes, and why? If for development purpose. Wrap this in condition.

Reply:
What?
If i don't enable CORES i can't do request

![image](https://user-images.githubusercontent.com/64608510/226114484-844c0a9f-dd19-4828-8e58-810f9254de2d.png)

You maybe made a mistake, with that idea, u think about something else

Reply:
If You want to fix this error You sould host Your enviroment on nginx or sth. With this Your frontend and backend will have same origin and this problem will be resolved. You can add this to docker easly. 

`{ cors: true }` is also good and way faster solution. But You enable this GLOBALY. Which is wrong because during deploy it's easy to forget to disable this option. So You should add condition to enable this only when its DEV mode.

Reply:
Done, i create value in ENV file